changelog/30265.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
+secrets/ldap: Update vault-plugin-secrets-openldap to v0.15.3
+```
```

Comment:
Instead of a `change` type with the version bump, I'd suggest a `bug` type that states what was fixed, along with user impact (kinda like we did in https://github.com/hashicorp/vault/pull/29174). The list of changes/fixes is small; no need to make folks hop through PRs and repo's to figure out what was fixed.

Reply:
thanks!